acceptance/run.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd $(dirname $0)/..
+build/builder.sh make install
```

Comment:
Placing the `certs` in the local directory allows the cockroach client to just work. Easy enough to put it in a temporary directory and provide an easy means of retrieving its location.